lib/actions/scan.coffee

```diff
@@ -32,6 +32,20 @@ dockerVersionProperties = [
 	'ApiVersion'
 ]
 
+scanErrorMessage = 'Could not find any balenaOS devices in the local network.'
+
+winScanErrorMessage = scanErrorMessage + """
+\n
+Note for Windows users:
+  The 'scan' command relies on the Bonjour service.
+  You can download Bonjour for Windows (included with Bonjour Print Services)
+	from one of these locations:
+	  https://bonjour.en.softonic.com/
```

Comment:
Can we trust `https://bonjour.en.softonic.com/` in relation to trojan, spyware, malware?... If unsure, I'd use the `apple.com` link only. Users can always google 'bonjour' and decide for themselves whether they want to risk installing something...